test/ApplicationInsightsJavaScriptSnippetTest/ApplicationInsightsJavaScriptSnippetTest.cs

```diff
@@ -78,8 +78,14 @@ public ApplicationInsightsJavaScriptSnippetTest(ITestOutputHelper output) : base
 
                     var validator = new Validator(httpClient, httpClientHandler, logger, deploymentResult);
 
-                    logger.LogInformation("Verifying home page");
-                    await validator.VerifyScriptCheckPage(response);
+                    logger.LogInformation("Verifying layout page");
+                    await validator.VerifyLayoutPage(response);
+
+                    logger.LogInformation("Verifying layout page before script");
+                    await validator.VerifyLayoutPageBeforeScript(response);
+
```

Comment:
Ah maybe that's best? Assuming they don't start encoding the key in some weird way, but that's a risk I'd be willing to take.